examples_tests/13.BRDF_Explorer/BRDFExplorerApp.cpp

```diff
@@ -395,15 +381,32 @@ void main() {
 		const float metallic = getMetallic(texCoords);
 		const vec3 baseColor = getAlbedo(texCoords);
 		const float ao = getAO(texCoords);
-
-        vec3 IoR_dielectric = getIoR_dielectric(texCoords);
-        mat2x3 IoR_conductor = mat2x3(getIoR_conductor(texCoords), uImagIoR);
+)"
+    +
+    [&_params] {
+    if (!_params.constantRI)
```

Comment:
!?

Reply:
hm?

Reply:
what is going on here?

Reply:
real ior is got in different way depending on whether it comes from texture or constant. And this code is generated in inline lambda (so that I don't have to create extra string for this code, imo it's more legible this way)